app/services/foreman_seeder.rb

```diff
@@ -11,7 +11,7 @@ class << self
   end
 
   def initialize
-    @seeds = (foreman_seeds + plugin_seeds).sort_by { |seed| seed.split("/").last }
+    @seeds = order_seeds(foreman_seeds) + order_seeds(plugin_seeds)
```

Comment:
I think we should keep the logic of sorting all plugins together with core, in case a plugin wants to run a seed interjecting between core seeds.
```suggestion
    @seeds = order_seeds(foreman_seeds + plugin_seeds)
```

Reply:
I see what you mean but that's a pretty significant divergence from the current behavior and arguably out of scope for this PR. Existing plugins may be reliant on those foreman seeds executing first as that's the existing behavior.

I checked luna-devel and it seems to be the case:

```
rake aborted!                                                                                                                                                                                                                                 
NoMethodError: undefined method `id' for nil:NilClass                                                                                                                                                                                         
/home/vagrant/foreman-tasks/app/models/foreman_tasks/task.rb:66:in `block in <class:Task>' 
<snip>
/home/vagrant/foreman-tasks/db/seeds.d/20-foreman_tasks_permissions.rb:8:in `<main>'   
```

That file is prefixed with 20 so the seeds didn't get very far at all, and it's probably not safe to assume that seeds across all plugins will actually raise an error due to the ordering change. They may silently continue without actually seeding anything.

Reply:
Generally, I am all for switching the sort to use numeric rather than lexicographic order. But I'm worried that it could also cause unexpected issues elsewhere do to the changed order, however we choose to tackle it.
If I look at the log snippet you shared previously, looks like at least the satellite theme does have a `069` seed, i'm not sure if this change would break that or not? Are there any other plugins that have a `0xx` seed that rely on that seed being executed in between foreman core seeds?